lib/runtime2/threepio/shared/api.ts

```diff
@@ -0,0 +1,502 @@
+export type ApiProvider = | 'qwen' | 'doubao';
+export type ApiConfiguration = ApiHandlerOptions & {
+  apiProvider?: ApiProvider;
```

Comment:
```suggestion
  provider: 'qwen' | 'doubao';
```
1. ApiProvider 没有必要再次定义，没有搜到其他地方有用到
2. api 语义重复了